RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/decorator/EventParam.kt

```diff
@@ -18,5 +18,7 @@ enum class EventParam(val paramName: String)
 	PUBLICATION_STRUCTURE_PATH("DV"),
 	SCREEN_SIZE("CS"),
 	WINDOW_SIZE("CV"),
-	CONSENT("_adpc")
+	CONSENT("_adpc"),
+	SELECTED_ELEMENT_NAME("VE"),
```

Comment:
to tu nie pasuje, to nie cześć dekoratora - zhardkoduj w factory np.

Reply:
Zastanawiałem się nad tym. Może lepiej klasę EventParam wynieść ponad pakiet dekoratora?
Jeśli te parametry będą w jednej klasie łatwiej będzie je odnaleźć i widać od razu jeśli ktoś chciałby przez przypadek powtórzyć nazwę parametru